src/test/components/__snapshots__/Footer.test.js.snap

```diff
@@ -50,7 +50,7 @@ exports[`Footer renders 1`] = `
   </nav>
   <p>
     Copyright Free Code Camp Columbus 2023 - 
-    2023
+    2024
```

Comment:
This will have to change in the future.  We cannot continue to change the snapshot every year.

Reply:
resolved for now, but we will have to change the snapshot for the test to be different so each year.  we need to find a better way to test